src/main/resources/init_db.sql

```diff
@@ -8,3 +8,30 @@ CREATE TABLE `taxi_service`.`manufacturers` (
 
 INSERT INTO `taxi_service`.`manufacturers` (`manufacture_name`, `manufacture_country`) VALUES ('Audi', 'Germany');
 INSERT INTO `taxi_service`.`manufacturers` (`manufacture_name`, `manufacture_country`) VALUES ('BMW', 'Germany');
+
+CREATE TABLE `cars` (
+                        `car_id` int NOT NULL AUTO_INCREMENT,
+                        `manufacturer_id` bigint DEFAULT NULL,
+                        `car_model` varchar(45) COLLATE utf8_bin DEFAULT NULL,
+                        `deleted` tinyint NOT NULL DEFAULT '0',
+                        PRIMARY KEY (`car_id`),
+                        KEY `manufacturer_id_idx` (`manufacturer_id`),
+                        CONSTRAINT `manufacturer_id` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`manufacture_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=73 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+CREATE TABLE `drivers` (
+                           `driver_id` int NOT NULL AUTO_INCREMENT,
+                           `driver_name` varchar(225) COLLATE utf8_bin NOT NULL,
```

Comment:
there is no need to indicate `driver_` prefix for each column because the columns are located in `drivers` table

same for other column names in manufacturer and car tables